src/commands/discord_service_manager.py

```
@@ -0,0 +1,161 @@
+        import requests
+        api_healthy = False
+        api_url = "http://localhost:8882/health"
+        
+        for _ in range(60):  # Try for 60 seconds
+            try:
+                response = requests.get(api_url, timeout=2)
+                if response.status_code == 200:
+                    api_healthy = True
+                    logger.info("API is healthy")
+                    break
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)